File: Models/EmailAccount.cs
using System;
using System.Collections.Generic;
using Microsoft.EntityFrameworkCore;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Newtonsoft.Json;

namespace AccountAPI.Models
{
	public class EmailAccount
    {
        [Key]
        public int EmailAccountId {get;set;}
        [Required]
        public string Email {get;set;}
        [Required]
        public string EmailPassword {get;set;}
        public DateTime CreatedAt {get;set;} = DateTime.Now;
        public DateTime UpdatedAt {get;set;} = DateTime.Now;

        public ICollection<Account> Accounts {get;} = new List<Account>();
    }
}

File: Client/src/app/Models/emailaccount.ts
export interface EmailAccountList {
  Id: number;
  Email: string;
  Password: string;
}

export interface EmailAccountAdd {
  Email: string;
  EmailPassword: string;
}

export interface EmailAccountEdit {
  EmailAccountId: number;
  Email: string;
  EmailPassword: string;
  UpdatedAt: Date;
}

export interface EmailAccount {
  EmailAccountId: number;
  Email: string;
  EmailPassword: string;
  CreatedAt: Date;
  UpdatedAt: Date;
  Accounts: EmailAccountAccounts[];
}

export interface EmailAccountAccounts {
  AccountId: number;
  Username: string;
  UserPassword: string;
}